File: app/src/main/java/in/woloo/www/refer_woloo_host/mvp/ReferredWolooView.java
package in.woloo.www.refer_woloo_host.mvp;

import in.woloo.www.more.models.ViewProfileResponse;
import in.woloo.www.refer_woloo_host.model.ReferredWolooListResponse;
import in.woloo.www.woloo_host.model.GeoCodeResponse;

public interface ReferredWolooView {
    void referredWolooListResponse(ReferredWolooListResponse referredWolooListResponse);
    void setProfileResponse(ViewProfileResponse viewProfileResponse);
    void referWolooHostSuccess(String message);
    void geoCodeResponseSuccess(GeoCodeResponse geoCodeResponse);

}

File: app/src/main/java/in/woloo/www/more/refer_woloo_host/mvp/ReferredWolooView.kt
package `in`.woloo.www.more.refer_woloo_host.mvp

import `in`.woloo.www.more.models.ViewProfileResponse
import `in`.woloo.www.more.refer_woloo_host.model.ReferredWolooListResponse
import `in`.woloo.www.more.woloo_host.model.GeoCodeResponse

interface ReferredWolooView {
    fun referredWolooListResponse(referredWolooListResponse: ReferredWolooListResponse?)
    fun setProfileResponse(viewProfileResponse: ViewProfileResponse?)
    fun referWolooHostSuccess(message: String?)
    fun geoCodeResponseSuccess(geoCodeResponse: GeoCodeResponse?)
}
